lib/mixpanel-node.d.ts

```diff
@@ -0,0 +1,159 @@
+    set(
+      distinctId: DistinctId,
+      propertyName: string,
+      value: string | number,
+      modifiers: Modifiers,
+      callback: ErrorCallback,
+    ): undefined;
+
+    set_once(distinctId: DistinctId, propertyName: string, value: string, callback?: ErrorCallback): undefined;
+    set_once(
+      distinctId: DistinctId,
+      propertyName: string,
+      value: string,
+      modifiers: Modifiers,
+      callback?: ErrorCallback,
+    ): undefined;
+    set_once(distinctId: DistinctId, properties: PropertyMap, callback?: ErrorCallback): undefined;
+    set_once(
+      distinctId: DistinctId,
+      properties: PropertyMap,
+      modifiers?: Modifiers,
+      callback?: ErrorCallback,
+    ): undefined;
+
+    increment(
+      distinctId: DistinctId,
+      propertyName: string,
+      modifiers?: Modifiers,
+      callback?: ErrorCallback,
+    ): undefined;
+    increment(
+      distinctId: DistinctId,
+      propertyName: string,
+      incrementBy: number,
+      modifiers: Modifiers,
+      callback?: ErrorCallback,
+    ): undefined;
+    increment(distinctId: DistinctId, propertyName: string, incrementBy: number, callback?: ErrorCallback): undefined;
+    increment(
+      distinctId: DistinctId,
+      properties: NumberMap,
+      modifiers: Modifiers,
+      callback?: ErrorCallback,
+    ): undefined;
+    increment(distinctId: DistinctId, properties: NumberMap, callback?: ErrorCallback): undefined;
+
+    append(
+      distinctId: DistinctId,
+      propertyName: string,
+      value: any,
+      modifiers: Modifiers,
+      callback?: ErrorCallback,
+    ): undefined;
+    append(distinctId: DistinctId, propertyName: string, value: any, callback?: ErrorCallback): undefined;
+    append(distinctId: DistinctId, properties: PropertyMap, callback?: ErrorCallback): undefined;
+    append(
+      distinctId: DistinctId,
+      properties: PropertyMap,
+      modifiers: Modifiers,
+      callback?: ErrorCallback,
+    ): undefined;
+
+    union(distinctId: DistinctId, data: UnionData, modifiers?: Modifiers, callback?: ErrorCallback): undefined;
+    union(distinctId: DistinctId, data: UnionData, callback: ErrorCallback): undefined;
+
+    track_charge(
+      distinctId: DistinctId,
+      amount: number | string,
+      properties?: PropertyMap,
+      callback?: ErrorCallback,
+    ): undefined;
+    track_charge(
+      distinctId: DistinctId,
+      amount: number | string,
+      properties: PropertyMap,
+      modifiers?: Modifiers,
+      callback?: ErrorCallback,
+    ): undefined;
+
+    clear_charges(distinctId: DistinctId, modifiers?: Modifiers, callback?: ErrorCallback): undefined;
+    clear_charges(distinctId: DistinctId, callback: ErrorCallback): undefined;
+
+    delete_user(distinctId: DistinctId, modifiers?: Modifiers, callback?: ErrorCallback): undefined;
+    delete_user(distinctId: DistinctId, callback: ErrorCallback): undefined;
```

Comment:
we should probably use `void` rather than `undefined`. If we are in-fact returning something object then it should be `any` or the type of the object.

Reply:
actually, looked at the source once again and it actually has a bunch of early `return;` statements. not sure if `undefined` or `void` is the way to go, then...